specification/security/_types/Privileges.ts

```diff
@@ -42,40 +42,100 @@ export class ApplicationPrivileges {
 export enum ClusterPrivilege {
   all,
   cancel_task,
+  /**
+   * @availability stack
+   */
   create_snapshot,
+  /**
+   * @availability stack
+   */
   grant_api_key,
   manage,
   manage_api_key,
+  /**
+   * @availability stack
+   */
   manage_ccr,
   manage_enrich,
+  /**
+   * @availability stack
+   */
   manage_ilm,
   manage_index_templates,
+  /**
+   * @availability stack
+   */
   manage_ingest_pipelines,
   manage_logstash_pipelines,
   manage_ml,
+  /**
+   * @availability stack
+   */
   manage_oidc,
   manage_own_api_key,
   manage_pipeline,
+  /**
+   * @availability stack
+   */
   manage_rollup,
+  /**
+   * @availability stack
+   */
   manage_saml,
   manage_security,
+  /**
+   * @availability stack
+   */
   manage_service_account,
+  /**
+   * @availability stack
+   */
   manage_slm,
+  /**
+   * @availability stack
+   */
   manage_token,
   manage_transform,
+  /**
+   * @availability stack
+   */
   manage_user_profile,
+  /**
+   * @availability stack
+   */
   manage_watcher,
   monitor,
   monitor_ml,
+  /**
+   * @availability stack
+   */
   monitor_rollup,
+  /**
+   * @availability stack
+   */
   monitor_snapshot,
   monitor_text_structure,
```

Comment:
I'm not seeing this one in https://github.com/elastic/elasticsearch-serverless/blob/c093830efcb3e2873e2307bb7b3da67d3966e768/modules/serverless-security/src/main/java/co/elastic/elasticsearch/serverless/security/privilege/ServerlessSupportedPrivilegesRegistry.java

Reply:
Great catch, I'll add a `@availability stack` here too. Thanks!